src/dredd-command.coffee

```diff
@@ -281,7 +296,8 @@ class DreddCommand
       return if @finished
 
     configurationForDredd = @initConfig()
-    @dreddInstance = @initDredd configurationForDredd
+    @logDebuggingInfo(configurationForDredd)
```

Comment:
Yes. It's poor man's implementation of #625 - with @netmilk we decided we'll just print this kind of info in the beginning of every Dredd run for now.